markdown/introduction.md

```diff
@@ -5,6 +5,18 @@ Nav: home.nav
 
 TableOfContents
 
+## {id="what"} What does it, really?
+
+What does it mean that "Bedrocklinux allows to run a linux system that is composed of different linux distributions"? It means, the system boots with a bootloader, a kernel, and an init system that can be independently selected from the "bedrocked" distributions, and can then freely run userland packages that are installable side-by-side from the different distributions.
+
+Consequently, the running system is rooted in different linux distributions.
+Therefore, a descriptive name for bedrocklinux concept could actually be multi-rooted-linux.
```

Comment:
The multi-root notion is an implementation detail that I don't think is suitable for introduction content.  While unlikely, in theory, it could change.  I think it better introduced along with other Bedrock-specific and release-specific concepts, which is what Bedrock's already documentation does.

Reply:
The "multi-root" implementation is kind of my personal favorite, idealized bedrock feature ;-)
Ideally, roots that may also be booted separately... :-)

Of course, the notion could go elsewhere. Thought that it could give a nice overall idea about the system, though.

Reply:
Shortening the sentence, but would want to keep "multi-rooted".
* it provides a understandable picture, even if not fully accurate to implementation details. 
* root(filesystem) and chroot has already has meaning to linux users (compared to unknown strata language)
```suggestion
Bedrocklinux could therefore be called a multi-rooted-linux.
```